File-Formats/DocIO/Working-with-Table-Of-Contents.md

```diff
@@ -581,10 +581,358 @@ document.Close();
 //https://help.syncfusion.com/file-formats/docio/create-word-document-in-xamarin#helper-files-for-xamarin
 {% endhighlight %}
 
-{% endtabs %}  
+{% endtabs %}
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Table-Of-Contents/TOC-with-user-defined-styles).
 
+## Applying switches for table of contents
+
+Switches are used to change the default settings in word document. TOC will determine the TOC entries based on the TOC switches. You can set heading levels, specify whether to use default heading styles, show page numbers, set page numbers to right alignment, set hyperlinks for TOC levels, preserve line break and text wrapping break in TOC, set outline levels for paragraph and specify whether to use table of entry fields. 
```

Comment:
You can modify the preservation of TOC entries in Word document by applying switches to table of contents. You can apply switches to preserve TOC entries as hyperlink or not, use default heading styles, preserve page numbers or not, aligning the page numbers, preserve new line characters, preserve outline level paragraphs, and preserve table of entry fields in the TOC entries using Word (DocIO) library.